app/models/topic_tracking_state.rb

```diff
@@ -389,34 +486,37 @@ def self.report_raw_sql(opts = nil)
     end
 
     sql = +<<~SQL
-    SELECT #{select}
-    FROM topics
-    JOIN users u on u.id = :user_id
-    JOIN user_stats AS us ON us.user_id = u.id
-    JOIN user_options AS uo ON uo.user_id = u.id
-    JOIN categories c ON c.id = topics.category_id
-    LEFT JOIN topic_users tu ON tu.topic_id = topics.id AND tu.user_id = u.id
-    LEFT JOIN category_users ON category_users.category_id = topics.category_id AND category_users.user_id = #{opts[:user].id}
-    LEFT JOIN dismissed_topic_users ON dismissed_topic_users.topic_id = topics.id AND dismissed_topic_users.user_id = #{opts[:user].id}
-    WHERE u.id = :user_id AND
-          #{filter_old_unread}
-          topics.archetype <> 'private_message' AND
-          ((#{unread}) OR (#{new})) AND
-          #{visibility_filter}
-          #{tags_filter}
-          topics.deleted_at IS NULL AND
-          #{category_filter}
-          NOT (
-            last_read_post_number IS NULL AND
-            COALESCE(category_users.notification_level, #{CategoryUser.default_notification_level}) = #{CategoryUser.notification_levels[:muted]}
-          )
-SQL
-
-    if opts[:topic_id]
+      SELECT #{select_sql}
+      FROM topics
+      JOIN users u on u.id = :user_id
+      JOIN user_stats AS us ON us.user_id = u.id
+      JOIN user_options AS uo ON uo.user_id = u.id
+      JOIN categories c ON c.id = topics.category_id
+      LEFT JOIN topic_users tu ON tu.topic_id = topics.id AND tu.user_id = u.id
+      LEFT JOIN category_users ON category_users.category_id = topics.category_id AND category_users.user_id = #{user.id}
+      LEFT JOIN dismissed_topic_users ON dismissed_topic_users.topic_id = topics.id AND dismissed_topic_users.user_id = #{user.id}
+      WHERE u.id = :user_id AND
+            #{filter_old_unread_sql}
+            topics.archetype <> 'private_message' AND
+            #{custom_state_filter ? custom_state_filter : "((#{unread}) OR (#{new})) AND"}
+            #{visibility_filter}
+            #{tags_filter}
+            topics.deleted_at IS NULL AND
+            #{category_filter}
+            NOT (
+              #{(skip_new && skip_unread) ? "" : "last_read_post_number IS NULL AND"}
+              (
+                COALESCE(category_users.notification_level, #{CategoryUser.default_notification_level}) = #{CategoryUser.notification_levels[:muted]}
+                AND tu.notification_level <= #{TopicUser.notification_levels[:regular]}
```

Comment:
Added this to the query so it matches up with what we are doing for muted categories in TopicQuery https://github.com/discourse/discourse/blob/2a8a23d9280c9bfd1af6976a301a4380d1c2adf9/lib/topic_query.rb#L899-L910

Otherwise, if someone has muted a category but has a tagged topic that is tracking or higher in that category, it will not show up in the totals.